Sprout/Views/TaskCell.m

```diff
@@ -28,7 +28,7 @@ - (void)refreshData {
 - (IBAction)didTapCompleted:(id)sender {
     PFUser *current = [PFUser currentUser];
     self.task.completed = !self.task.completed;
-    if(self.task.completed) {
+    if (self.task.completed) {
         current[@"completedTasks"] =  [NSNumber numberWithInt:[current[@"completedTasks"] intValue] + 1];
```

Comment:
If you have time, you could make models in Xcode for your classes so you can access and/or modify attributes using dot notation (since it looks cleaner) instead of brackets